packages/yum.go

```diff
@@ -131,7 +131,7 @@ func parseYumUpdates(data []byte) []*PkgInfo {
 			}
 			break
 		}
-		pkgs = append(pkgs, &PkgInfo{Name: string(pkg[0]), Arch: osinfo.Architecture(string(pkg[1])), Version: string(pkg[2])})
+		pkgs = append(pkgs, &PkgInfo{Name: string(pkg[0]), Arch: OSArch{StandardArch: osinfo.Architecture(string(pkg[1])), RawArch: string(pkg[1])}, Version: string(pkg[2])})
```

Comment:
Line become to long, that might be a good idea to separate structure construction and changing slice